qiskit/execute_function.py

```diff
@@ -149,15 +150,34 @@ def execute(experiments, backend,
 
         seed_simulator (int): Random seed to control sampling, for when backend is a simulator
 
-        default_qubit_los (list): List of default qubit LO frequencies in Hz
+        default_qubit_los (Optional[List[float]]): List of job level qubit drive LO frequencies
+            in Hz. Overridden by ``schedule_los`` if specified. Must have length ``n_qubits.``
 
-        default_meas_los (list): List of default meas LO frequencies in Hz
+        default_meas_los (Optional[List[float]]): List of job level measurement LO frequencies in
+            Hz. Overridden by ``schedule_los`` if specified. Must have length ``n_qubits.``
 
-        schedule_los (None or list or dict or LoConfig): Experiment LO
-            configurations, if specified the list is in the format::
+        qubit_lo_range (Optional[List[List[float]]]): List of job level drive LO ranges each of form
+            ``[range_min, range_max]`` in Hz. Used to validate ``qubit_lo_freq``. Must have length
+            ``n_qubits``.
 
-                list[Union[Dict[PulseChannel, float], LoConfig]] or
-                     Union[Dict[PulseChannel, float], LoConfig]
+        meas_lo_range (Optional[List[List[float]]]): List of job level measurement LO ranges each of
+            form ``[range_min, range_max]`` in Hz. Used to validate ``meas_lo_freq``. Must have
+            length ``n_qubits``.
+
+        schedule_los (Optional[Union[List[Union[Dict[PulseChannel, float], LoConfig]],
+                               Union[Dict[PulseChannel, float], LoConfig]]]):
+            Experiment level (ie circuit or schedule) LO freq configurations for qubit drive and
+            measurement channels. These values override the job level values from
+            ``default_qubit_los`` and ``default_meas_los``. Frequencies are in Hz. Settable for qasm
+            and pulse jobs.
+
+            If a single LO config or dict is used, the values are set at job level If a list is
```

Comment:
```suggestion
            If a single LO config or dict is used, the values are set at job level. If a list is
```

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/6167/commits/705ca3c97ae84c6f950f430f1eea576d4eb93e1c